code/src/GridAPI/ColumnManager.ts

```diff
@@ -163,4 +163,32 @@ namespace GridAPI.ColumnManager {
             'ColumnManager.destroyColumn-end'
         );
     }
+
+    /**
+     * Set column aggregate in group panel
+     *
+     * @export
+     * @param {string} gridID ID of the Grid where the change will occur.
+     * @param {string} columnID id of the column with which actions on the column can be performed.
+     * @param {wijmo.Aggregate} aggregate aggregate that will be applied on group panel.
+     */
+    export function SetColumnAggregate(
+        gridID: string,
+        columnID: string,
+        aggregate: wijmo.Aggregate
```

Comment:
Why are we passing an wijmo as an input to the API?

Reply:
opa, nice catch!

Reply:
fixed